lib/ansible/modules/cloud/misc/virt.py

```diff
@@ -586,18 +587,18 @@ def main():
     )
 
     if not HAS_VIRT:
-        module.fail_json(msg='The `libvirt` module is not importable. Check the requirements.')
+        module.fail_json(msg="The 'libvirt' module is not importable. Check the requirements.")
 
     rc = VIRT_SUCCESS
     try:
         rc, result = core(module)
     except Exception as e:
-        module.fail_json(msg=to_native(e), exception=traceback.format_exc())
```

Comment:
preserving the actual traceback in `exception` is usually a thing we want to keep (so you get the "if you want to see the full traceback pass -vvv" message)

Reply:
Not really, Ansible adds these automatically these days. So we don't have to add them individually.